### PR TITLE
feat(replays): add banner about masking config docs to breadcrumbs tab

### PR DESCRIPTION
**Pull request by michellewzhang**
- Shows for every user on the replay details breadcrumb tab
- Is dismissible, only comes back if local storage is cleared
- Links to https://docs.sentry.io/platforms/javascript/session-replay/privacy/

SCR-20231219-oqpr

Closes https://github.com/getsentry/team-replay/issues/322

### CODE REVIEW

**Review by c298lee — APPROVED** [review]

**Comment by ryan953** [comment]
Can we style the Alert to remove the margin along the bottom?

File: static/app/views/replays/detail/breadcrumbs/index.tsx [hunk 1]
```diff
@@ -126,6 +134,19 @@ function Breadcrumbs() {
       <FilterLoadingIndicator isLoading={isFetchingExtractions || isFetchingTraces}>
         <BreadcrumbFilters frames={frames} {...filterProps} />
       </FilterLoadingIndicator>
+      {isDismissed ? null : (
+        <Alert
+          type="info"
+          showIcon
+          trailingItems={<StyledCloseButton onClick={dismiss} />}
```
Comment (ryan953) [inline static/app/views/replays/detail/breadcrumbs/index.tsx:141]: This is not a11y friendly. 

I think `<Button icon={<IconClose />} onClick={dismiss} />` is the right combo.

Comment (ryan953) [reply static/app/views/replays/detail/breadcrumbs/index.tsx:142]: with whatever the correct `size=` if needed

File: static/app/views/replays/detail/breadcrumbs/index.tsx [hunk 2]
```diff
@@ -172,4 +203,8 @@ function Breadcrumbs() {
   );
 }
 
+const StyledAlert = styled(Alert)`
```
Comment (michellewzhang) [inline static/app/views/replays/detail/breadcrumbs/index.tsx:206]: still need some margin here otherwise it touches the top of the table

**Review by ryan953 — APPROVED** [review]

**Pull request merged**